operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -737,6 +741,10 @@ public boolean keyReplaced() {
         return renewalType == RenewalType.REPLACE_KEY;
     }
 
+    public boolean keyCreated() {
```

Comment:
`isKeyCreated` would be slightly more idiomatic, but given the other similar methods are not using `is` I can understand why you matched the convention